tests/test_backend.py

```diff
@@ -20,9 +20,14 @@
 from __future__ import unicode_literals
 
 import onnx.backend.test
+import pytest
 
 from ngraph_onnx.onnx_importer.backend import NgraphBackend
 
+backend_name = pytest.config.getoption('backend', default='CPU')
```

Comment:
This variable is redundant, you can `NgraphBackend.backend_name = pytest.config` directly